packages/nss.rb

```diff
@@ -3,70 +3,64 @@
-     armv7l: '812f36d5c1875cf8803097265483b8a558194c6f0778977d55637407682ba51c',
-       i686: 'e724b986a9a2edce790a8bef0b89d096ff46cb93815fc2d7bf8beb950af851f5',
-     x86_64: 'ba50721b2e968c49fbd5d4a90112ff27374903d33dbcfc3f1877b2c7074adbcd'
+    aarch64: 'e68a620a7b7f35289f401561e3c86ff1fceb38d0b0652da7e6c531addc020a14',
+     armv7l: 'e68a620a7b7f35289f401561e3c86ff1fceb38d0b0652da7e6c531addc020a14',
+       i686: '88d3a49f76d9fe8722aab3f9553a576609741004c2acdd2773bef28b2cc9e12a',
+     x86_64: '89f80e57d77168fe58e28f60e245a18256dd3b1c68640a1430fc512f93e01b01'
   })
 
   depends_on 'gyp_next' => :build
-  depends_on 'nspr'
 
   def self.build
-    ENV['opt_build'] = '1'
-    ENV['build_64'] = if ARCH == 'x86_64'
-                        '1'
-                      else
-                        '0'
-                      end
-    @arch_cflags = if ARCH == 'armv7l'
-                     ''
-                   else
-                     '-flto=auto'
-                   end
+    @build_64 = ARCH == 'x86_64' ? '1' : '0'
+    @arch_cflags = ''
     @arch_ldflags = @arch_cflags
 
-    ENV['NS_USE_GCC'] = '1'
-    ENV['CPPFLAGS'] = "-I#{CREW_PREFIX}/include/nspr"
-    ENV['USEABSPATH'] = 'NO'
-    ENV['NSS_GYP_PREFIX'] = CREW_PREFIX
     Dir.chdir 'nss' do
-      system "env CFLAGS='-pipe #{@arch_cflags}' \
+      system "env opt_build=1 build_64=#{@build_64} \
+        NSS_ENABLE_WERROR=0 NS_USE_GCC=1 USEABSPATH=NO \
+        NSS_GYP_PREFIX=#{CREW_PREFIX} CFLAGS='-pipe #{@arch_cflags}' \
         CXXFLAGS='-pipe #{@arch_cflags}' LDFLAGS='#{@arch_ldflags}' \
         ./build.sh -v --opt --gcc --gyp \
-        --with-nspr=#{CREW_PREFIX}/include/nspr --system-nspr \
         --system-sqlite --disable-tests"
     end
+    system "sed -i \
+    -e 's,^libdir=.*,libdir=#{CREW_LIB_PREFIX},g' \
+    -e 's,^prefix=.*,prefix=#{CREW_PREFIX},g' \
+    -e 's,\${exec_prefix}/lib,\${exec_prefix}/#{ARCH_LIB},g' \
+    dist/Release/lib/pkgconfig/nspr.pc"
   end
 
   def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/include/nss"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/include/nspr"
     FileUtils.rm Dir.glob('dist/Release/lib/*.so.TOC')
     FileUtils.mv 'dist/Release/lib', "dist/Release/#{ARCH_LIB}" unless ARCH_LIB == 'lib'
     FileUtils.cp_r Dir.glob('dist/Release/*'), CREW_DEST_PREFIX
     FileUtils.cp_r Dir.glob('dist/public/nss/*'), "#{CREW_DEST_PREFIX}/include/nss/"
-
+    FileUtils.cp_r Dir.glob('dist/public/nspr/*'), "#{CREW_DEST_PREFIX}/include/nspr/"
     system "sed nss/pkg/pkg-config/nss.pc.in \
     -e 's,%libdir%,#{CREW_LIB_PREFIX},g' \
     -e 's,%prefix%,#{CREW_PREFIX},g' \
-    -e 's,%exec_prefix%,#{CREW_PREFIX}/bin,g' \
+    -e 's,%exec_prefix%,#{CREW_PREFIX},g' \
     -e 's,%includedir%,#{CREW_PREFIX}/include/nss,g' \
-    -e 's,%NSPR_VERSION%,$(pkg-config --modversion nspr),g' \
-    -e 's,%NSS_VERSION%,#{@_ver},g' | \
+    -e 's,%NSPR_VERSION%,#{@nspr_ver},g' \
+    -e 's,%NSS_VERSION%,#{@nss_ver},g' | \
```

Comment:
Redirect to file instead of using install command?

Reply:
Just suggest the replacement code. :)

Reply:
Like this
`sed … > #{CREW_DEST_LIB_PREFIX}/pkgconfig/nss.pc`

Reply:
Can fix this on the next version?  I don't want to build the i686 binary again.  It takes forever. :)

Reply:
It doesn’t require a rebuild 😅

Reply:
But a true test would require it. ;)